packages/table/src/table.tsx

```diff
@@ -177,6 +177,14 @@ export interface ITableProps extends IProps, IRowHeights, IColumnWidths {
     onCopy?: (success: boolean) => void;
 
     /**
+     * A callback called when the visible cell indices change in the table.
+     */
+    onVisibleCellsChange?: (rowStartIndex: number,
```

Comment:
this might be hard to read in the docs. alternative is to define a type alias for this function, but we don't currently document those =(